### ISSUE TITLE
at-rule-no-unknown :Unexpected unknown at-rule "@use"

### ISSUE THREAD

**Issue by maxjf1** [kristerkari/stylelint-scss#511]
The rule `at-rule-no-unknown` complains abouse the [`@use`](https://sass-lang.com/documentation/at-rules/use) rule, wich is a default rule in sass/scss.

## Versions:
```
  stylelint: 8.4.0
  stylelint-config-standard: 17.0.0
  stylelint-scss: 2.5.0
```

## .stylelintrc.json:
```json
{
    "extends": "stylelint-config-standard",
    "plugins": [
        "stylelint-scss"
    ],
    "rules": {
        "at-rule-no-unknown": null,
        "scss/at-rule-no-unknown": true
    }
}
```

**Comment by kristerkari** [comment]
It should handle the use of `@use`. Can you give an example of a code that throws an error?

**Comment by maxjf1** [comment]
`_functions.scss`
```scss
@use "variables";
```
output:
```bash
scss/_functions.scss
 1:1  ✖  Unexpected unknown at-rule "@use"                     scss/at-rule-no-unknown
         (at-rule-no-unknown)
```

**Comment by dsimon-onventis** [comment]
I'm using the `@use` rule in my projects, and it does not show an error.

I recognized you are using an old version of everything. Did you try updating all the dependencies to their latest version?

**Comment by dsimon-onventis** [comment]
You are using an old version of the package. The support of `@use` was introduced in [Release 3.12.0](https://github.com/kristerkari/stylelint-scss/releases/tag/3.12.0). Update to this version (or later) to fix the issue.

**Comment by kristerkari** [comment]
oh yeah, I did not even check the version because I was expecting the latest one to be used :) Thanks @dsimon-onventis! 

@maxjf1 Just updated to the latest version and the error goes away.
